src/app/_components/FrontLayout/FrontLayout.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Grid, Typography, Box, Divider, useMediaQuery } from "@mui/material";
+import Logo from "~/assets/images/logo_font.svg";
+import Image from "next/image";
+import theme from "~/theme";
+
+const FrontLayout = (): JSX.Element => {
+  const mobileSmallScreen = useMediaQuery(theme.breakpoints.down("sm"));
+
+const MobileFooterContainer = () => {
+  return (
+    <Box id="have-an-account" sx={{
+      textAlign: "center",
+      marginTop: "auto",
+      marginBottom: "1rem",
+      position: "absolute",
+      bottom: "1rem",
+      width: mobileSmallScreen ? "100vw" : "100%",
+      borderTop: "1px solid #CCC",
+      paddingTop: "13px",
+      height: mobileSmallScreen ? "10px" : "40px",
+    }}
+    > Have an account? Container
+    </Box>
+  )
+}
```

Comment:
You file doesn't look formatted, can you try again? Make sure prettier is enabled

Reply:
um I think I found a solution, will see if it works